model/project_parser.go

```diff
@@ -486,7 +486,7 @@ func (pss *parserStringSlice) UnmarshalYAML(unmarshal func(interface{}) error) e
 
 // LoadProjectForVersion returns the project for a version, either from the parser project or the config string.
 // If read from the config string and shouldSave is set, the resulting parser project will be saved.
-func LoadProjectForVersion(v *Version, id string, shouldSave bool) (ProjectInfo, error) {
+func LoadProjectForVersion(v *Version, id string) (ProjectInfo, error) {
```

Comment:
Can we audit usages of this function? I'm wondering if (now that we don't need version config) we can just use a smaller function to get and translate the parserProject (and not the projectRef and config) in some places (and not have to look up version since we could just use the id)

Reply:
Looks like `createVersionForTag` is the only invocation that needs the `Ref` and `Config` fields to be set on the `ProjectInfo`.  I've refactored most usages to use a new function `FindAndTranslateProjectForVersion` that will as you suggest, only get and translate the parser proj without checking for project ref or project config (reducing DB calls). I'm keeping this function here but it now uses `FindAndTranslateProjectForVersion` as a helper.

Reply:
Awesome! Can we rename this function to LoadProjectInfoForVersion to distinguish it more?